internal/pkg/rpaas/k8s.go

```diff
@@ -133,44 +142,171 @@ func (q *fixedSizeQueue) Next() *remotecommand.TerminalSize {
 	return q.sz
 }
 
-func (m *k8sRpaasManager) Exec(ctx context.Context, instanceName string, args ExecArgs) error {
-	instance, err := m.GetInstance(ctx, instanceName)
+func (m *k8sRpaasManager) Debug(ctx context.Context, instanceName string, args DebugArgs) error {
+	instance, debugContainerName, status, err := m.debugPodWithContainerStatus(ctx, &args, instanceName)
 	if err != nil {
 		return err
 	}
+	if status.State.Terminated != nil {
+		req := m.kcs.CoreV1().Pods(instance.Namespace).GetLogs(args.Pod, &corev1.PodLogOptions{Container: debugContainerName})
+		return logs.DefaultConsumeRequest(req, args.Stdout)
```

Comment:
It's possible I overlooked something however I really didn't get it... Why should we see the logs if pod is the termination phase?

Reply:
Cause it's possible to send a command and not enter in interact mode with debug container. In this case scenario container enter in terminated state and code throw to user container logs related to the command sent (this feature its similar how kubectl debug works in this scenario)